sdk/tdf.go

```diff
@@ -322,6 +331,14 @@ func (r *Reader) Manifest() Manifest {
 // prepare the manifest for TDF
 func (s SDK) prepareManifest(ctx context.Context, t *TDFObject, tdfConfig TDFConfig) error { //nolint:funlen,gocognit // Better readability keeping it as is
 	manifest := Manifest{}
+
+	version, err := ParseVersion(sdkVersion)
```

Comment:
We need the ability to extend this version based on the decision of going with option 2 in this adr.

https://github.com/opentdf/platform/issues/1677

Reply:
https://github.com/opentdf/platform/pull/1805/files#diff-e4ef1c45819a3bdf9ccddb7d496711ecc1275b3c2ee9155c6273597283f74cc1R1496 - will handle the option 2